Tools/TestWebKitAPI/Tests/WebKitCocoa/PermissionsAPI.mm

```diff
@@ -102,4 +124,44 @@ static void urlEncodeIfNeeded(uint8_t byte, StringBuilder& buffer)
     EXPECT_FALSE(didReceiveQueryPermission);
 }
 
+TEST(PermissionsAPI, PermissionMayHaveChanged)
+{
+    [TestProtocol registerWithScheme:@"https"];
```

Comment:
Is this needed?

Reply:
The way I had written the request in this iteration of the patch--this line was needed. But there's a better way to write the request that doesn't require this. I'll use that better version in the next iteration of this patch.